src/Pagination/Pagination.scss

```diff
@@ -57,6 +57,7 @@
 
 .page-link {
   border: none;
+  background-color: transparent;
```

Comment:
[curious] Is this property overriding an existing `background-color` property that's coming from somewhere? I see in the inspector that `.page-link` has `background-color: #fff;`, which is overridden by this line.

Should we be relying more on the stock button variant styles in `Pagination` instead? E.g., now that we're pulling in the "tertiary" button variant, that already has a transparent background on it.

I believe introducing this property here may be overriding the default hover/focus styles for the buttons in `Pagination` as well (e.g., there are no hover styles on the tertiary buttons in the [inverse pallete](https://deploy-preview-1849--paragon-openedx.netlify.app/components/pagination/#inverse-pallete-default-size)).

Reply:
yeah, I think you're right, we'll take a closer look at this